2.4/alpine/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM alpine:3.4
+
+# add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
+RUN addgroup -g 82 -S www-data \
+	&& adduser -u 82 -D -S -G www-data www-data
```

Comment:
I think it'd be good to mimic the block we've got in PHP: https://github.com/docker-library/php/blob/e37ca400ba0e1c34357bf06732bc77064e5a4941/Dockerfile-alpine.template#L17-L24 (which will also make it clear why we chose to hard-code `82` here)

```dockerfile
# ensure www-data user exists
RUN set -x \
	&& addgroup -g 82 -S www-data \
	&& adduser -u 82 -D -S -G www-data www-data
# 82 is the standard uid/gid for "www-data" in Alpine
# http://git.alpinelinux.org/cgit/aports/tree/main/apache2/apache2.pre-install?h=v3.3.2
# http://git.alpinelinux.org/cgit/aports/tree/main/lighttpd/lighttpd.pre-install?h=v3.3.2
# http://git.alpinelinux.org/cgit/aports/tree/main/nginx-initscripts/nginx-initscripts.pre-install?h=v3.3.2
```